hail/src/main/scala/is/hail/io/vcf/LoadVCF.scala

```diff
@@ -1081,22 +1081,6 @@ case class MatrixVCFReader(
 
   override lazy val fullType: TableType = LowerMatrixIR.loweredType(fullMatrixType)
 
-  val partitioner = if (partitionsJSON != null) {
```

Comment:
what's this about?

Reply:
Just saw some weird dead code and deleted it

Reply:
The point of this code was to have an analogue to the `partitionsJSON` in `VCFsReader`, but it is dead now. It would be good to give this kind of control of partitioning at some point, but there's always just using `import_vcfs` instead.